docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -30,6 +30,7 @@ Some stable features are enabled by default. You can disable a stable feature by
 | `internationalization`              | Enables internationalization                                                         | Yes                |
 | `accessTokenExpirationCheck`        | Enable OAuth access_token expiration check and token refresh using the refresh_token |                    |
 | `disablePrometheusExemplarSampling` | Disable Prometheus examplar sampling                                                 |                    |
+| `logsSampleInExplore`               | Access to logs sample feature in Explore                                             | Yes                |
```

Comment:
```suggestion
| `logsSampleInExplore`               | Enables access to the logs sample feature in Explore                                             | Yes                |
```

Reply:
@ivanahuckova , does that ^^ wording work? Want to make sure I'm not changing the technical meaning. Thank you.

Reply:
Yes it does, but I had there `Enables logs sample feature in Explore` before and it got override by linting and changed it to `Access to logs sample feature in Explore`. Do you want me to try what you have suggested, to see if linter will be okay with it?

Reply:
Fixed in https://github.com/grafana/grafana/pull/62291/commits/1c0ddf27c786cc6f97dae2dd6bc7d4441de40b8a. Thank you so much for review and suggestion!